guide/14-deep-learning/FCN_MLModel_guide-final_draft.ipynb

```diff
@@ -0,0 +1,501 @@
+{
```

Comment:
<p>(1) The two <s>methodology</s> <strong>methodologies </strong>for modelling are as follows.</p><p>(2) In the second <strong>option,</strong> any machine learning algorithm from <s>scikit learn</s> <strong>scikit-learn </strong>can be implemented</p><p>(3) ML algorithm from the <strong>scikit-learn</strong> library <s>just by passing</s>  <strong>by just passing</strong> the name of the algorithm</p><p>(4) This <s>tools</s><strong> tool </strong>makes it possible for model selection among the several machine learning models available in <strong>scikit-learn</strong></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/718/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>